app/controllers/currency.js

```diff
@@ -9,52 +9,48 @@ var timestamp = +new Date(),
 
 exports.index = function(req, res) {
 
-  var _xhr = function() {
-    if (typeof XMLHttpRequest !== 'undefined' && XMLHttpRequest !== null) {
-      return new XMLHttpRequest();
-    } else if (typeof require !== 'undefined' && require !== null) {
-      var XMLhttprequest = require('xmlhttprequest').XMLHttpRequest;
-      return new XMLhttprequest();
-    }
-  };
-
-  var _request = function(url, cb) {
-    var request;
-    request = _xhr();
-    request.open('GET', url, true);
-    request.onreadystatechange = function() {
-      if (request.readyState === 4) {
-        if (request.status === 200) {
-          return cb(false, request.responseText);
+    var _xhr = function() {
+        if (typeof XMLHttpRequest !== 'undefined' && XMLHttpRequest !== null) {
+            return new XMLHttpRequest();
+        } else if (typeof require !== 'undefined' && require !== null) {
+            var XMLhttprequest = require('xmlhttprequest').XMLHttpRequest;
+            return new XMLhttprequest();
         }
+    };
 
-        return cb(true, {
-          status: request.status,
-          message: 'Request error'
-        });
-      }
+    var _request = function(url, cb) {
+        var request;
+        request = _xhr();
+        request.open('GET', url, true);
+        request.onreadystatechange = function() {
+            if (request.readyState === 4) {
+                if (request.status === 200) {
+                    return cb(false, request.responseText);
+                }
+
+                return cb(true, {
+                    status: request.status,
+                    message: 'Request error'
+                });
+            }
+        };
+
+        return request.send(null);
     };
 
-    return request.send(null);
-  };
-
-  // Init
-  var currentTime = +new Date();
-  if (bitstampRate === 0 || currentTime >= (timestamp + delay)) {
-    timestamp = currentTime;
-
-    _request('https://www.cryptonator.com/api/ticker/aur-usd', function(err, data) {
-      if (!err) bitstampRate = parseFloat(JSON.parse(data)['ticker']['price']);
-
-      res.jsonp({
-        status: 200,
-        data: { bitstamp: bitstampRate }
-      });
-    });
-  } else {
-    res.jsonp({
-      status: 200,
-      data: { bitstamp: bitstampRate }
-    });
-  }
-};
+    // Init
+    var currentTime = +new Date();
+    if (bitstampRate === 0 || currentTime >= (timestamp + delay)) {
+        timestamp = currentTime;
+        bitstampRate = 1.337;
```

Comment:
If the currency conversion is broken, can we disable whatever is using it instead of returning a hard-coded exchange rate?

Reply:
If this really is bitstamp, then we should probably change it to coinmarketcap or something similar -- and in any case prefer a slightly older exchange rate rather than a completely bogus one. 
The SMLY price is eg here: curl https://chainz.cryptoid.info/smly/api.dws?q=ticker.usd
and the conversion to other currencies can be from lots of places, eg
curl https://api.exchangeratesapi.io/latest?base=USD&symbols=ISK